src/molecules/formfields/CreditCardField/index.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from 'atoms/Icon';
+
+import formStyles from 'molecules/formfields/shared/formfields.module.scss';
+import creditCardStyles from './credit_card_field.module.scss';
+
+const styles = Object.assign({}, formStyles, creditCardStyles);
+
+function CreditCardField(props) {
+  const {
+    className,
+    label,
+    input,
+    meta,
+    cardType,
+  } = props;
+
+  const baseClass = [
+    styles['formfield'],
+    styles['credit-card'],
+    { [styles.focused]: meta && meta.active },
+    className
+  ];
+
+  return (
+    <div className={classnames(...baseClass)}>
```

Comment:
Makes sense, I haven't figured out the cleanest way to allow the organizing of components.